middlewares/auth.middleware.js

```diff
@@ -0,0 +1,15 @@
+const { ERRORS } = require('../constants');
+const { extractToken } = require('../services/user.service');
+const { HttpStatusCode } = require('axios');
+
+const authMiddleware = (req, res, next) => {
+  const token = extractToken(req);
+  if (token == null) {
+    return res.status(HttpStatusCode.Unauthorized).send(ERRORS.UNAUTHORIZED);
+    ks;
+  }
```

Comment:
What is this `ks` ?

Reply:
its an accident keyboard type, I removed it